spec/slack-ruby-bot/hooks/message_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+
+class DummyBot < SlackRubyBot::Bot; end
```

Comment:
Declaring classes inside specs has a nasty side effect of instantiating these globally. So while this declaration "seems" local, it's not. `DummyBot` now exists elsewhere and any re-declaration of `DummyBot` reopens this class and adds to it. It's still manageable here, but becomes really problematic in larger projects.

Two ways to address it:

* Namespace the bot, for example:

```ruby
module MessageSpec
  class DummyBot ...
  end
end
```
This is a cop-out :)

* Declare support classes in spec/support/... If `DummyBot` were to be used in multiple specs, I would do that (doesn't seem like the case here).

* Create classes dynamically inside the spec with `let`. If `DummyBot` is only to be used in this spec, do that.

```ruby
let(:dummy_bot) {
  Class.new(SlackRubyBot::Bot) do
  end
}
```
I would do the latter here.